packages/shadydom/src/patch-events.js

```diff
@@ -44,7 +44,7 @@ const supportsEventOptions = (() => {
 
 const parseEventOptions = (optionsOrCapture) => {
   let capture, once, passive, shadyTarget;
-  if (optionsOrCapture && typeof optionsOrCapture === 'object') {
+  if (optionsOrCapture && optionsOrCapture instanceof Object) {
```

Comment:
I would do:

```suggestion
  if (optionsOrCapture && (typeof optionsOrCapture === 'object' || typeof optionsOrCapture === 'function')) {
```

because `new Boolean(false) instanceof Object` is true, and to keep the same cross-realm behavior. Plus `typeof` is very fast compared to `instanceof`.

Reply:
`new Boolean(false) == false` so it wouldn't get past the `&&` here at all.

We can put back the `typeof` as a perf optimization.

Reply:
I just mean that any `new {Primitive}()` will return an object, `new Boolean(true)` too. That could conceivably flow into options.